mvt/android/cli.py

```
@@ -54,9 +55,12 @@ def download_apks(all_apks, virustotal, koodous, all_checks, output, from_file):
         if from_file:
             download = DownloadAPKs.from_json(from_file)
         else:
-            if not output:
-                log.critical("You need to specify an output folder (with --output, -o) when extracting APKs from a device")
-                sys.exit(-1)
+            try:
+                os.makedirs(output)
+            except OSError as e:
+                if e.errno != errno.EEXIST:
```

You should probably re-throw the error again if it's not handled here